### DISCUSSION TITLE
Too easy to overwrite Title field in table view

### DISCUSSION THREAD

**Discussion by gold2718** [community/community#15815] [Projects and Issues]
This issue is about the table view of the new Beta projects board.

If you accidentally hit any key while your cursor is in the Title field, you instantly change the title field which causes several issues:
- There is no undo button
- It changes the title of the associated issue or PR with no confirmation (unlike the title from the issue or PR page where you have to click both edit and save to change it).
- If the line is not yet an issue or PR, there is no way to recover the old text (see no undo button above).

Some way to protect text fields would be nice!

**Comment by LosAhuertos** [comment]
I want to look at the clothesless and tea on the webcam

пт, 29 апр. 2022 г., 22:18 goldy ***@***.***>:
